lib/puppet/type/rabbitmq_cluster.rb

```diff
@@ -40,6 +40,11 @@
     end
   end
 
+  newparam(:local_node) do
+    desc 'Name of the local node'
+    defaultto('')
```

Comment:
I'm a bit rusty on Puppet types, but could this default to `nil` or be unset, or does it have to be an empty string?

Reply:
fixed